s3watcher/deploy.json

```diff
@@ -13,11 +13,19 @@
                     }
                 }
             }
+        },
+        "s3-watcher": {
+            "type": "aws-s3",
+            "data": {
+                "bucket": "media-service-dist",
+                "cacheControl": "public, max-age=60",
+                "publicReadAcl": false
```

Comment:
Although secrets are obtained from the bucket that triggers the lambda, setting `publicReadAcl` to false as paranoid...